test-utils/src/main/java/io/littlehorse/test/LHExtension.java

```diff
@@ -107,7 +107,6 @@ private void maybeCreateTenantAndPrincipal(TestContext testContext) {
                     .putPrincipal(PutPrincipalRequest.newBuilder()
                             .setId(principalId)
                             .setGlobalAcls(acls)
-                            .putPerTenantAcls(testContext.getConfig().getTenantId(), acls)
```

Comment:
No need to do this anymore. This makes the default principal a Tyler for end-to-end tests.